### ISSUE TITLE
[BUG] Quick Fix not showing up

### ISSUE THREAD

**Issue by aaronpowell** [microsoft/vscode-dotnettools#1610]
### Describe the Issue

I'm using the [Public API Analyzer](https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md) in https://github.com/CommunityToolkit/Aspire but since adding .NET 9 to our devcontainer the analyzer quick fix is no longer showing up (but it was prior to the .NET 9 release/adding .NET 9 to the devcontainer).

It's detected:

![Image](https://github.com/user-attachments/assets/e28d1828-76e9-41fb-b7fd-95abeca63874)

But not an option:

![Image](https://github.com/user-attachments/assets/867f5307-8a17-487d-b0cc-42b980523818)

The feature works in VS though.

### Steps To Reproduce

1. Clone https://github.com/CommunityToolkit/Aspire and open in the devcontainer
2. Add a new public method to any of the classes in projects under the `src` folder
3. Wait for the analyzer to run
4. Activate the quick fix menu

### Expected Behavior

I'd expect that the Analyzer code fix option to be presented and execute, as it did prior to updating.

### Environment Information

- OS: devcontainer
- C# DevKit version: 1.14.2 (pre-release)
- VS Code version:

```
Version: 1.96.0-insider (system setup)
Commit: db865c38f0f5fee1de1653b75dd66ec802d24a28
Date: 2024-11-13T17:56:05.737Z
Electron: 32.2.1
ElectronBuildId: 10427718
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.26120
```

**Comment by svick** [comment]
I suspect this is a duplicate of https://github.com/dotnet/vscode-csharp/issues/7802.

**Comment by dibarbet** [comment]
yup, should be fixed in 2.58.20 (prerelease)

**Comment by PedroContipelli** [comment]
Still have this issue on latest version. No quick fixes show up for me. Running VSCode in WSL and a .NET 8 project
